packages/kobber-components/src/progress-bar/ProgressBarItem.js

```diff
@@ -0,0 +1,68 @@
+import "@gyldendal/kobber-base/themes/default/tokens.css";
+
+/**
+ * @summary An autonomous custom progress bar element.
+ *
+ * The progress bar is not native <progress> elements, due to poor
+ * tweakability of the native element (both semantically and styling).
+ *
+ */
+export class ProgressBarItem extends HTMLElement {
+  constructor() {
+    super();
+    this.attachShadow({ mode: "open" });
+    this.fillColorFallback = "--kobber-component-progressbar-color-foreground-default-primary";
+    this.heightValueFallback = "var(--kobber-component-progressbar-color-background-default)";
```

Comment:
Hvorfor er `var()` med her, men ikke i linja over? Mulig jeg overser noe?

Reply:
Ah nei, det er en forglemmelse - det skal være `var()` overalt! Det bryter med min foreslåtte konvensjon: "pass around full css values (not just custom property names)".
Tanken er at verdien med `var()` rundt enkelt kan erstattes av en hvilken som helst css-verdi (f eks fra json). Bruker vi ikke `var()`, så må mer av koden skrives om dersom vi ønsker å erstatte verdiene med noe annet enn css-variablene.